pymobiledevice3/cli/mounter.py

```diff
@@ -157,17 +156,31 @@ def mounter_auto_mount(lockdown: LockdownClient, xcode: str, version: str):
     signature = f'{image_path}.signature'
     developer_disk_image_dir = Path(image_path).parent
 
-    if not developer_disk_image_dir.exists():
-        try:
-            download_developer_disk_image(version, developer_disk_image_dir)
-        except PermissionError:
-            logger.error(
-                f'DeveloperDiskImage could not be saved to Xcode default path ({developer_disk_image_dir}). '
-                f'Please make sure your user has the necessary permissions')
-            return
-
     image_path = Path(image_path)
     signature = Path(signature)
+
+    available_versions = get_all_versions()
+
+    if version not in available_versions:
+        logger.error(f"Unable to find DeveloperDiskImage for {version}. available versions: {available_versions} ")
```

Comment:
- change to single quote
- remove trailing space

Reply:
k